gtsfm/scene_optimizer.py

```diff
@@ -112,7 +114,10 @@ def create_computation_graph(
         image_graph: List[Delayed],
         camera_intrinsics_graph: List[Delayed],
         image_shape_graph: List[Delayed],
-        gt_cameras_graph: Optional[List[Delayed]] = None,
+        relative_pose_priors: Dict[Tuple[int, int], Delayed],
+        absolute_pose_priors: List[Delayed],
+        gt_cameras_graph: List[Delayed],
+        gt_poses_graph: List[Delayed],
```

Comment:
the pose is already in the camera, so you would not need an additional argument for it I think.

Reply:
Its tricky to handle that if some cameras are None. I had to handle helper functions to extract poses. For now, having a dedicated pose graph is a stopgap.

Reply:
we have to revisit this, we dont need gt_cameras then, and only gt_calibration would suffice? I think @travisdriver unified them in some PR?